torchvision/models/detection/backbone_utils.py

```diff
@@ -124,7 +125,8 @@ def _resnet_fpn_extractor(
 
     if returned_layers is None:
         returned_layers = [1, 2, 3, 4]
-    assert min(returned_layers) > 0 and max(returned_layers) < 5
+    if min(returned_layers) <= 0 or max(returned_layers) >= 5:
+        raise ValueError(f" `returned_layers` object should contain integers between [1,4], got {returned_layers} ")
```

Comment:
Maybe more idiomatic is 
```suggestion
        raise ValueError(f"Each returned layer must be in the range [1,4]. Got {returned_layers}")
```